src/ApplicationResolver.php

```diff
@@ -66,10 +66,12 @@ protected function getEnvironmentSetUp(\Illuminate\Foundation\Application $app):
      * @param  string $class
      *
```

Comment:
Maybe we can add here `@phpstan-param class-string $class` 

There is an error in TravisCI about this.

Reply:
What's the difference between the two?

Reply:
PHPStan supports a special string type. `class-string` its the fully qualifed class name. For example `User::class` And it uses this in the places that you expect not just any string but a class name. Like in `ReflectionClass` here.

Reply:
It seems this is not enough. `$class` should be documented as `class-string` also where the `isServiceProvider` method is called.

Reply:
That's up on line 46 of ApplicationResolver - I'm not sure how to annotate the use there as you're wanting to.

Reply:
You can try to put the docblock on line 42 maybe `@var class-string $class`

Reply:
Ok, that's up, but I still had larastan complain when running locally, on line 46 of ApplicationResolver:
  46     Parameter #1 $class of static method NunoMaduro\Larastan\ApplicationResolver::isServiceProvider() expects class-string, string given.  

`

Reply:
Removing it from line 42 and putting it right above the `return substr($class, 0, strlen($namespace)) === $namespace && self::isServiceProvider($class);` line, works :+1:

Reply:
It should be `/**` With 2 *. That way PHPStan can recognize it.

Reply:
Thanks for your help and patience, @canvural .  I've sent your suggested change up after confirming analyse runs clean locally.